Week3/js-exercises/ex5-LotteryMachine.js

```diff
@@ -27,12 +27,39 @@ Don't you just love the thrill of the lottery? What if I told you we can make ou
 */
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
-  const numbers = [];
-  // make array
+  let numbers = [];
+  // make array => re-initiate the variable numbers to new array
+  numbers = new Array();
+
+  // 1- iterate through the array using startIndex and stopIndex to generate array:
+  for (let i = 0; i < stopIndex; i++) {
+    startIndex++;
+    numbers[i] = startIndex;
+    if (numbers[i] === stopIndex) {
+      // break the loop when numbers element reach to the stopIndex
+      break;
+    }
+  }
```

Comment:
you made this a bit too complicated
```suggestion
  for (let i = startIndex; i <= stopIndex; i++) {
    numbers.push(i);
  }
```

Reply:
Thank you for the suggestion. It was really complicated with my code. Sometimes, things are easy, but I could not think well to solve them.